Orange/widgets/visualize/utils/owlegend.py

```diff
@@ -232,9 +233,10 @@ def __init__(self, color, parent):
         self.__square.setBrush(QBrush(color))
         self.__square.setPen(QPen(QColor(0, 0, 0, 0)))
         self.__square.setParentItem(self)
```

Comment:
Sorry, after `self.__square` is defined, but _before_ `self` is set as its parent.